UI/setup/upgrade_info.html

```diff
@@ -6,10 +6,9 @@
 <div class="setupconsole">
 <h2><?lsmb text('Database Management Console') ?></h2>
 <div class="listtop"><?lsmb text('Upgrade Info') ?></div>
-<form data-dojo-type="dijit/form/Form"
+<form data-dojo-type="lsmb/setup/upgrade_info"
```

Comment:
We have "data migration pre-checks" in the form of our Upgrade_Tests. Why are we suddenly taking a radically different approach? Is it impossible to handle the situation in an Upgrade_Test?

Reply:
This only thing this does is reflect the form input validity onto the submit button. This isn't yet at Upgrade_Test level, and should really have been handled by Dojo. When user inputs are coherent with fields restrictions, we should validate the Upgrade_Test.

Reply:
But ... we don't do that anywhere else in the application... starting to do so here right now will result in an inconsistent UI experience. Is that what we want?

Reply:
On the contrary, each field where we added constraints do prevent using the submit button. They just do not reflect it on the button state,  but refuse the submit when clicked with invalid fields on the form.

Reply:
Agreed: clicking the button will show which fields are blocking submission. It's like that in the entire application. Why should we use this change as an opportunity to create inconsistencies?

Reply:
Why do we impose waiting for the user to click the submit button instead of handling the form validity. We know that the form is invalid but do not tell and force the user to ask. I say that this should be reflected elsewhere instead of mainting the current behaviour.

Reply:
I'm not saying we want to maintain the current behaviour. I'm saying if we want to change it, we need to do so everywhere. And a change like that needs its own PR. All I'm saying is that is not good to be inconsistent and that changing the behaviour is not part of the topic for this PR.